.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,59 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: bug
+---
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/buidler-aragon/blob/master/CONTRIBUTING.md#ways-to-contribute)?
+
+- [ ] Yes
+
+## To Reproduce
+
+<!-- Write your steps here: -->
+
+1. In aragon app '...'
+2. Run command '....'
+3. Then did '....'
+4. See error
+
+## Expected behavior
+
+<!--
+  How did you expect your project to behave?
+  It’s fine if you’re not sure your understanding is correct.
+  Just write down what you thought would happen.
+-->
+
+(Write what you thought would happen.)
+
+## Actual Behavior
+
+<!--
+  Did something go wrong?
+  Is something broken, or not behaving as you expected?
+  Describe this section in detail, and attach screenshots if possible.
+  Don't just say "it doesn't work"!
+-->
+
+(Write what happened. Add screenshots, if applicable.)
+
+## Context
+
+#### Network and buidler.config?
+
+(Which network you are using the app with.)
+
+#### Aragon App project?
```

Comment:
```suggestion
#### Aragon app project?
```

Apparently the official term is "Aragon app" (without the second capitalized a) :).

Reply:
Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474